eng/Versions.props

```diff
@@ -144,7 +144,7 @@
     <SystemIdentityModelTokensJwtPackageVersion>5.3.0</SystemIdentityModelTokensJwtPackageVersion>
     <WindowsAzureStoragePackageVersion>8.1.4</WindowsAzureStoragePackageVersion>
     <!-- Dependencies for Blazor. -->
-    <MicrosoftAspNetCoreBlazorMonoPackageVersion>0.8.0-preview-20190125.1</MicrosoftAspNetCoreBlazorMonoPackageVersion>
+    <MicrosoftAspNetCoreBlazorMonoPackageVersion>0.8.0-preview-20190204.1</MicrosoftAspNetCoreBlazorMonoPackageVersion>
```

Comment:
Out of curiosity, where does this package get built? We should figure out a way to move it to some place we can get automated signing

Reply:
The sources are in https://github.com/aspnet/blazor, and it gets built in the blazor-ci official build pipeline. It already gets signed there.